pages/common/find.md

```diff
@@ -33,4 +33,4 @@
 
 - Find empty files (0 byte) or directories and delete them verbosely:
 
-`find {{root_path}} -type {{f|d}} -empty -delete -print`
+`find {{root_path}} -type '{{f,d}}' -empty -delete -print`
```

Comment:
This example is not wrong :D I'd rather leave it like this. The `|` is fine, because you either have a `f` (for file) or a `d` for directory there.

Reply:
The `|` is tldr syntax (similar to `{{}}`), so users would change the command accordingly to either contain `-type f` or `-type d`. 

However, I realise that the current form of the command is indeed misleading. I would wait what others have to say on this issue.

Reply:
No problem - though I think the example text `Find empty files (0 byte) or directories and delete them verbosely` is pretty specific and doesn't make sense without the corrected syntax

Reply:
What was the event that made you feel that this command needed to be changed? I want to understand if it was a clarity issue or if it's just misunderstanding for not knowing the syntax.

Reply:
What was the event that made you feel that this command needed to be changed? I want to understand if it was a clarity issue or if it's just misunderstanding for not knowing the syntax.

Reply:
What was the event that made you feel that this command needed to be changed? I want to understand if it was a clarity issue or if it's just misunderstanding for not knowing the syntax.

Reply:
The command for the example is wrong - the syntax is wrong in two ways - I've fixed it to run and correctly work for the  described purpose

Reply:
![image](https://github.com/user-attachments/assets/5f493b44-5580-4555-a02a-44e41b93b777)
Works fine for me.

Reply:
it skips directories

Reply:
Well yes, the description does say "or". It can be interpretted in two ways I guess. I'll ruminate on this for a while and try to come up with a a better wording. I do think f|d is the way to go, just that the wording needs to be more precise that the user needs to choose.